### PR TITLE
DEVDOCS-4511: [remove] Orders v2, remove form_fields from Create an Order

### PR DESCRIPTION
**Pull request by bc-tgomez**
# [DEVDOCS-4511]

## What changed?
* Remove `form_fields` from POST request
* Remove additional description from the `form_field` values

## Anything else?
Related PRs, salient notes, etc

[DEVDOCS-4511]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4511?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by bc-tgomez** [comment]
Heya @donald-nguyen-bc I reached out in Slack for some feedback on the description changes. Would you like to see anything else in this PR?

File: reference/orders.v2.oas2.yml [hunk 1]
```diff
@@ -4075,11 +4063,11 @@ components:
           example: '0.0000'
           type: string
         base_wrapping_cost:
-          description: 'The value of the base wrapping cost expressed as a floating point number to four decimal places in string format.'
+          description: The value of the base wrapping cost expressed as a floating point number to four decimal places in string format.
           example: '0.0000'
           type: string
         billing_address:
-          $ref: '#/components/schemas/billingAddress_Full'
+          $ref: '#/components/schemas/billingAddress_Base'
```
Comment (donald-nguyen-bc) [inline reference/orders.v2.oas2.yml:4070]: may you pls add (in the PR description) some stoplight studio screenshots of:
- the billing_address in POST/PUT request payload does not contain the `form_field` field
- the billing_address in GET response contains the `form_field` field as read-only

I doubt changing this `order_Shared` will remove the `form_field` from the GET response as well.

Comment (markcmurphy) [reply reference/orders.v2.oas2.yml:4070]: @donald-nguyen-bc, I assisted @bc-tgomez with this, so I am chiming in:

`POST` `v2/orders` and  `PUT` `v2/orders/{order_id}`: 
image

`GET` `v2/orders`: 
image

Comment (donald-nguyen-bc) [reply reference/orders.v2.oas2.yml:4070]: awesome @markcmurphy, it looks good

**Review by donald-nguyen-bc — APPROVED** [review]
👏

**Review by stanzikratelbc — APPROVED** [review]
LGTM. Ready to merge.

**Pull request merged**